packages/schemas/src/MosaicConfigSchema.ts

```diff
@@ -1,10 +1,12 @@
 import { z } from 'zod';
 import { fileExtensionSchema } from './fileExtensionSchema.js';
+import { modeSchema } from './modeSchema.js';
 import { pluginModuleSchema } from './PluginModuleSchema.js';
 import { serialiserModuleSchema } from './SerialiserModuleSchema.js';
 import { sourceModuleSchema } from './SourceModuleSchema.js';
 
 export const mosaicConfigSchema = z.object({
+  deployment: z.object({ mode: modeSchema, platform: z.string() }).optional(),
```

Comment:
I guess we could be opinionated on platform values , rather than just string ?

Reply:
fixed